README.md

```diff
@@ -1,6 +1,9 @@
 # Deprecated
 
-This project is officially deprecated in favor of [Docker Machine](https://docs.docker.com/machine/).  The code and documentation here only exist as a reference for users who have not yet switched over (but please do soon).
+This project is officially deprecated in favor of [Docker
+Machine](https://docs.docker.com/machine/).  The code and documentation here
+only exist as a reference for users who have not yet switched over (but please
+do soon).
```

Comment:
Probably should also mention Docker Toolbox like the updated fallback message does.